packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -361,22 +364,31 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 
     private async handleSummaryAcks() {
         let refSequenceNumber = this.runtime.deltaManager.initialSequenceNumber;
+        let ack: IAckedSummary | undefined;
         while (this.runningSummarizer) {
             const summaryLogger = this.runningSummarizer.tryGetCorrelatedLogger(refSequenceNumber) ?? this.logger;
             try {
-                const ack = await this.summaryCollection.waitSummaryAck(refSequenceNumber);
+                ack = await this.summaryCollection.waitSummaryAck(refSequenceNumber);
```

Comment:
Please add this above this line:
```ts
ack = undefined;
```

This will ensure that if exception happens inside of waitSummaryAck on second iteration, we record undefined, not previous handles.